manifests/elasticsearch.pp

```diff
@@ -17,7 +17,7 @@
   $elasticsearch_path_logs = '/var/log/elasticsearch',
 
 
-  $elasticsearch_ip = 'localhost',
+  $elasticsearch_ip = '<Put Elasticsearch IP>',
```

Comment:
Please change this to `<YOUR_ELASTICSEARCH_IP>` as we have in our Chef repo.

Reply:
Hi Jose!

Thanks for your attention. Check the changes [here](https://github.com/wazuh/wazuh-puppet/commit/1301d408b350d150f40ea1e58dbace59119cc99a).

Kind regards,

Rshad